src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,219 @@
+       :text-with-one-icon   [page-nav-mid-comp {:text-color                       text-color
+                                                 :mid-section-main-text            mid-section-main-text
+                                                 :align-left?                      one-icon-align-left?
+                                                 :mid-section-icon                 mid-section-icon
+                                                 :mid-section-main-text-icon-color mid-section-main-text-icon-color}]
+       :text-with-description [rn/view (assoc centrify
+                                              :flex-direction    :row
+                                              :margin-horizontal 2)
+                               (when (and mid-section-description-user-icon
+                                          (not horizontal-description?))
+                                 [rn/image {:source {:uri mid-section-description-user-icon}
+                                            :style  {:width             32
+                                                     :height            32
+                                                     :border-radius     32
+                                                     :margin-horizontal 6}}])
+                               [rn/view {:style {:flex-direction (if horizontal-description?
+                                                                   :row
+                                                                   :column)}}
+                                [rn/text {:size   :base
+                                          :weight :font-semi-bold
+                                          :style  {:color      text-color
+                                                   :text-align :left}}
+                                 mid-section-main-text]
+                                (when mid-section-description
+                                  [rn/view {:style {:flex-direction :row}}
+                                   (when (and mid-section-description-icon
+                                              (not horizontal-description?))
+                                     [icons/icon mid-section-description-icon
+                                      (icon-props text-secondary-color :small)])
+                                   [rn/text {:size   :small
+                                             :weight :font-medium
+                                             :style  {:padding-right 4
+                                                      :margin-left   2
+                                                      :text-align    :left
+                                                      :line-height   18
+                                                      :color         text-secondary-color}}
+                                    mid-section-description]])]])]))
+
+(defn right-section-icon
+  [props {:keys [bg icon icon-color]}]
```

Comment:
consider being more verbose with variable names. 
e.g here it could be `background-color` instead of `bg`. The saved characters do not add much value and just lead to potential confusion.

Reply:
Understood, Will be as clear as I can be.
Changed